tasks/bot-avatars.md

```diff
@@ -131,7 +131,7 @@ on things from this list.  Be creative!
 - sales bot (bot that plugs into your Etsy/Squarespace/Shopfiy/Stripe/Square/etc
   eCommerce site or payments system and updates the chat every time an item is
   sold or refunded)
-- revenue bot (bots that plug into your eCommerce site and updates the chat on
+- revenue bot (bots that plugs into your eCommerce site and updates the chat on
```

Comment:
Your change is incorrect. "plugs" as a verb would work if the subject were singular, as in:

"my bot plugs into a site"

But here, the subject ("bots") is plural, so the verb should be "plug".